modules/core/src/main/scala/io/ergolabs/cardano/explorer/core/db/sql/OutputsSql.scala

```diff
@@ -261,7 +261,7 @@ final class OutputsSql(implicit lh: LogHandler) {
            |  o.value,
            |  encode(o.data_hash, 'hex'),
            |  case when (d.value is null) then rd.value else d.value end,
-           |  encode(rd.raw_value, 'hex'),
+           |  case when (d.bytes is null) then encode(rd.raw_value, 'hex') else encode(d.bytes, 'hex') end,
```

Comment:
For now we don't use `reported_datum` table. Probably we should drop it from all queries